cypress/integration/view-component-metadata.e2e.js

```diff
@@ -36,5 +36,71 @@ describe('Viewing component metadata', () => {
         .last()
         .should('have.text', ' Change Detection Strategy: Default');
     });
+
+    it('should display correct set of inputs', () => {
```

Comment:
Can we also add tests for updating the properties?

Reply:
Those are already in property-update and property-edit tests. https://github.com/rangle/angular-devtools/tree/main/cypress/integration

Reply:
I was thinking of updating renamed inputs to verify both reads and writes work.